charts/backstage/Chart.yaml

```
@@ -44,4 +44,4 @@ sources:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 3.3.1
+version: 3.3.3
```

```suggestion
version: 3.4.0
```

This is a new feature, IMO, but still backward-compatible.